nuget/bootstrap.nuspec

```diff
@@ -2,16 +2,16 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
   <metadata>
     <id>bootstrap</id>
-    <version>4.4.1</version>
+    <version>4</version>
```

Comment:
What's the rationale behind not using the full version here?

Reply:
I  added a comment to the file; the version is pulled from package.json so no need to maintain this